grammar/liquid-html.ohm

```diff
@@ -97,6 +98,9 @@ Liquid <: Helpers {
   liquidTagSection = liquidTagRule<"section", liquidTagSectionMarkup>
   liquidTagSectionMarkup = liquidString space*
 
+  liquidTagSections = liquidTagRule<"sections", liquidTagSectionsMarkup>
+  liquidTagSectionsMarkup = liquidString space*
```

Comment:
It's indeed always a string? Never anything else?

Reply:
always a string